data/src/main/java/com/globant/equattrocchio/data/response/Image.java

```diff
@@ -4,18 +4,16 @@
 import com.google.gson.annotations.SerializedName;
 
 public class Image {
```

Comment:
If this class a response for a service, you need change the name for ImageResponse